lms/djangoapps/learner_dashboard/views.py

```diff
@@ -45,3 +45,20 @@ def view_programs(request):
     }
 
     return render_to_response('learner_dashboard/programs.html', context)
+
+
+@login_required
+@require_GET
+def program_details(request, programId):
```

Comment:
Pylint is not a fan camel-cased variable names. `programId` should be `program_uuid`.